src/store.js

```diff
@@ -0,0 +1,5 @@
+import { Provider } from 'react-redux';
+import { createStore } from "redux";
+
+
+export default store;
```

Comment:
Also we don't appear to be connecting the store to anything, or creating it anywhere.  Normally that is the sort of thing which happens in the entry point (which looks like index.js, i guess?)

Reply:
Yes, we can have it in entry point but some redux tutorials had a separate folder for store too. I couldn't figure out which is a better way so I just kept it this way.